### ISSUE TITLE
XCode build error

### ISSUE THREAD

**Issue by LocalGod79** [endless-sky/endless-sky#1346]
```
Undefined symbols for architecture x86_64:
  "Conversation::DIE", referenced from:
      ___cxx_global_var_init in Conversation.o
  "Conversation::FLEE", referenced from:
      ___cxx_global_var_init in Conversation.o
  "Conversation::DEFER", referenced from:
      ___cxx_global_var_init in Conversation.o
  "Conversation::ACCEPT", referenced from:
      ___cxx_global_var_init in Conversation.o
  "Conversation::DEPART", referenced from:
      ___cxx_global_var_init in Conversation.o
  "Conversation::LAUNCH", referenced from:
      ___cxx_global_var_init in Conversation.o
  "Conversation::DECLINE", referenced from:
      ___cxx_global_var_init in Conversation.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by MarcelineVQ** [comment]
This seems to be caused by Conversation::ACCEPT, Conversation::DECLINE, etc, not being declared in Conversation.cpp despite being defined in Converstion.h

Simply adding:
```
const int Conversation::ACCEPT;
const int Conversation::DECLINE;
const int Conversation::DEFER;
const int Conversation::LAUNCH; // Accept + leave immediately.
const int Conversation::FLEE; // Decline + leave immediately. 
const int Conversation::DEPART; // Defer + leave immediately. 
const int Conversation::DIE;
```
to Conversation.cpp's main namespace is enough to fix it but I don't know if that's the right solution or just 'a' solution, I'm not well versed in linker issues.

**Comment by endless-sky** [comment]
Fixed in dc260a3.
